### DISCUSSION TITLE
I cannpt find MARKOR on GooglePlayStore

### DISCUSSION THREAD

**Discussion by pussstudio** [gsantner/markor#1990] [Q&A] [answered]
I cannpt find MARKOR on GooglePlayStore since abt. 3 weeks before.

What's up?

**Comment by harshad1** [comment]
Hmmm. I can't either.

**Comment by gsantner** [comment, marked as answer]
Hello, the main source of Markor was always either of F-Droid or directly here from GitHub. You can install from there, thank you.

Google Play is quite time consuming and prefer to have the time for improving the code :-).

**Reply by jonzyk** [reply]
Hi, first of all, your tool is great thank you.
My company phone locks all external app store.
I had finally found a solution for my personnal notes with Markor + Syncthing. I hope I can keep it like that for some time and Markor will not be automaticaly removed from my phone.

**Comment by t-arn** [comment]
I don't like alternative Stores. Can't you publish Markor in Google Play again, please? Because I really like it but I won't install it from alternative stores
Best regards

**Reply by gsantner** [reply]
To everyone on it's own, you can also directly download and install Markor from [here](https://github.com/gsantner/markor/releases).

I don't plan to publish Markor on Google Play.

> Because I really like it but I won't install it from alternative stores

If you want to have it, you have a way to install it. If thats not an option to you, I'm sorry.

**Reply by ichmoimeyo** [reply]
I also like and use Markor on my Android phones.  I'm aware  of your policy but find it unfortunate.
Like many and a growing number of Chromebook users I cannot install Markor in a regular way via the allowed Google  Play store. 
I either need to put my Chromebook into the less secure "Developer mode" or install Linux(Crostini).
I just mainly post this in case you are not aware of  this - I am not affected as one of my Chromebooks is in Developer mode and on the other one I still have an old version from the Play store.
Best regards and  thanks for an excellent program that [I keep recommending](https://forum.obsidian.md/t/use-joplin-on-android-to-edit-vault/60141/3?u=ichmoimeyo) to many Obsidian Mobile Android users as some of them find making quick notes too slow.

**Comment by DS-109** [comment]
Welcome back at Google's Play Store!
This prevents getting trapped by TMarkor and other cloners.

**Reply by harshad1** [reply]
Please let me know how I can help with play store stuff (writing emails, making changes to meet some absurd requirement). I know that maintaining Markor on the play store can be hard.

**Reply by o-kotb** [reply]
I can't find Markor in Japanese play store.

**Reply by gsantner** [reply]
It's not on Google Play currently. Might also stay as that, as Google constantly changes their mind, blocks meaningful things or requires more data.

Get Markor from F-Droid or GitHub, the preffered source.

**Reply by konkun11** [reply]
Hi!
So Markor won't be available in Play Store anymore?

I've restored my phone and couldn't find the app :(
